src/AudioPasteDialog.h

```diff
@@ -0,0 +1,40 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file AudioPasteDialog.h
+
+  @author Vitaly Sverchinsky
+
+**********************************************************************/
+
+#pragma once
+
+#include "widgets/wxPanelWrapper.h"
+
+class wxCheckBox;
+class ShuttleGui;
+
+class AudioPasteDialog final : public wxDialogWrapper
+{
+   DECLARE_DYNAMIC_CLASS(WaveClipCopyWarningDialog)
```

Comment:
This macro isn't needed and the named class doesn't exist

Reply:
Thanks!